streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordCollectorImpl.java

```diff
@@ -212,11 +212,11 @@ public <K, V> void send(final String topic,
                     keyClass,
                     valueClass),
                 exception);
-        } catch (final SerializationException exception) {
+        } catch (final Exception exception) {
```

Comment:
@mjsax - I guess the intention is to catch all other errors during serialization, not just the serialization exception. I think this is fine, I don't know if there are exceptions but I guess not: seems like the handler should either indicating there's an error and rethrow or continue.